content/02.introduction.md

```diff
@@ -5,7 +5,7 @@
 Most scientists, including ecologists and evolutionary biologists, are increasingly dependent on computational tools in their research [@doi:10.1109/SECSE.2009.5069155].
 Researchers write and use code as part of their scientific workflow to perform a wide-variety of tasks ranging from data management, data analysis, and study replication, to the application and the development of tools for hypothesis testing.
 To maintain code for scientific collaboration requires an efficient and well-documented work-flow [@doi:10.1038/d41586-020-02462-7].
-To facilitate this process, scientists have been increasingly adopting tools from information and systems technology, such as cloud-based services for documentation and version control (e.g., from the Google Suite, with Docs, Sheets, and Drive; the Microsoft Suite, with Word, Excel, and OneDrive; and GitHub; @doi:10.1038/538127a).
+To facilitate this process, scientists have been increasingly adopting tools from information and systems technology, such as cloud-based services for documentation and version control (_e.g._, from the Google Suite, with Docs, Sheets, and Drive; the Microsoft Suite, with Word, Excel, and OneDrive; and GitHub; @doi:10.1038/538127a).
```

Comment:
Do we need the spaces in "e.g."? It looks a little odd - but I guess will render differently? Should we change this throughout the MS?

Reply:
Hi, @DrMattG. The markdown writing `(_e.g._,` translates to "(_e.g._,", i.e. it italicizes "e.g.,". It is a good question indeed if this should be done across the entire paper. I have seen journal author guidelines recommending both ways. I have italicized every entry I have found until this edit, but I can also remove this formatting if necessary.

Reply:
This is a style thing that varies from journal to journal and will probably get done by a copy editory.  I wouldn't worry about it either way at this point.

Reply:
I think generally we left out italics in the manuscript.